frontend/src/main/scala/bloop/testing/LoggingEventHandler.scala

```diff
@@ -111,6 +102,22 @@ class LoggingEventHandler(logger: Logger) extends TestSuiteEventHandler {
     case TestSuiteEvent.Done => ()
   }
 
+  private def extractErrors(events: List[Event], logger: Logger) =
+    events
+      .filter(e => failedStatuses.contains(e.status()))
```

Comment:
Why do we need to put `failedStatuses` in an object? I think it's used only here.